tests/stress.rs

```diff
@@ -4701,6 +4703,14 @@ fn run_stress_tests() {
                 .unwrap()
         );
 
-        stress_functions(&ctx)
+        ctx
+    }
+
+    #[test]
+    fn run_stress_tests() {
+        unsafe {
+            let ctx = create_context();
+            stress_functions(&ctx);
```

Comment:
@dignifiedquire The tests fail when I move out the code to `create_context()` but it compiles. Wondering what the difference is.

Reply:
Still wondering what I'm doing wrong here, since it compiles but tests are failing.